include/psa/crypto.h

```diff
@@ -4009,6 +3974,565 @@ psa_status_t psa_generate_key(const psa_key_attributes_t *attributes,
 
 /**@}*/
 
+
+/** \defgroup Password-authenticated key agreement
```

Comment:
I guess then I should move everything PAKE related into `crypto_extra.h`.

Reply:
Yes, I think that's easy to do and clarifies our intentions. That plus adding a note at the top of the PAKE section, because `crypto_extra.h` also contains things that are Mbed TLS extensions but for which we do promise stability.